components/brave_wallet_ui/utils/amount.ts

```diff
@@ -387,4 +394,4 @@ export default class Amount {
 
     return formatter.format(value) + abbreviation
   }
-}
+}
```

Comment:
nit: missing EOF new-line

Reply:
Fixed in [603c091](https://github.com/brave/brave-core/pull/24328/commits/603c091556c832696d0e0c499ab984bcdffd97cb)